tests/phpunit/TemplateTest.php

```diff
@@ -41,8 +41,8 @@ protected function process_page($text) {  // Only used if more than just a citat
 
   public function testDisableParallelBuilds() { // This test should be first one in file
      if (PHP_VERSION_ID < 60000) {  // Either 5.* or 7.*
-         echo "\n Sleeping PHP 6 build for 60 seconds \n";
-         sleep(60);  // Delay the non-7 build for 30 seconds to keep us from hitting the API wall
+         echo "\n Sleeping PHP 6 build for 120 seconds \n";
```

Comment:
If APIs have such a large window, then they don't sound scaleable: are they really designed to be used in a bot context?

Reply:
I have been avoiding it, but I can get 10 Keys easily and will wrote code to rotate through them.

I have been waiting for google to up my quota on this key

Reply:
Array of ten keys by the end of the weekend :-(